src/content/en/2019/media.md

```diff
@@ -129,93 +243,207 @@ While previously it appeared that GIF files were smaller than JPEG, we can now c
 
-## Accessibility
+<figure>
+  <iframe aria-labelledby="fig17-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://todo"></iframe>
+  <a href="/static/images/2019/04_Media/fig17.png" class="fig-mobile">
+    <img src="/static/images/2019/04_Media/fig17.png" aria-labelledby="fig17-caption" width="600">
+  </a>
+  <div id="fig7-caption" class="visually-hidden">A comparison of the Lighthouse Audit for Offscreen content</div>
+  <figcaption>Figure 17. Lighthouse Audit: Offscreen</figcaption>
+</figure>
+
+The Lighthouse audit provides us a smell as there are a number of situations that can provide tricky to detect such as the use of quality placeholders. 
+
+LazyLoading [can be implemented](https://developers.google.com/web/fundamentals/performance/lazy-loading-guidance/images-and-video) in many different ways including using a combination of Intersection-Observers or Resize-Observers or using javascript libraries like [lazySizes](https://github.com/aFarkas/lazysizes), [lozad](https://github.com/ApoorvSaxena/lozad.js), and a host of others. 
+
+In 2019-August, Chrome 76 launched with the support for markup based lazy loading using `<img loading=”lazy”>`. While the snapshot of websites used for the 2019 Web Almanac used 2019-July, over 2,509 websites already utilized this feature.
+
+### Accessibility
 
 At the heart of image accessibility is the alt tag.  When the alt tag is added to an image, this text can be used to describe the image to a user who is unable to view the images (either due to a disability, or a poor internet connection.  
 
 We can detect all of the image tags in the HTML files of the dataset. Of 13M image tags on desktop and 15M on mobile, 91.6% of images have an alt tag present.  At initial glance, it appears that image accessibility is in very good shape on the web.  However, upon deeper inspection, the outlook is not as good.  If we examine the length of the alt tags present in the dataset, we find that the median length of the alt tag is 6 characters.  This maps to an empty alt tag (appearing as alt=””).  Only 39% of images use alt text that is longer than 6 characters.  The median value of ‘real’ alt text is 31 characters  - of which 25 actually describe the image.
 
-# Video
+## Video
 While images dominate the media being served on web pages, videos are beginning to have a major role in content delivery on the web.  According to HTTP Archive, we find that 4.06% of desktop and 2.99% of mobile sites are self hosting video files (meaning that the files are not hosted at websites like YouTube or Facebook.).
 
+### Video Formats
+
 Video can be delivered with many different formats, and players.  The dominant formats for mobile and desktop are .ts (segments of HLS streaming) and .mp4 (the H264 MPEG):
- (figure (count of video files by extension)
+
+<figure>
+  <iframe aria-labelledby="fig19-caption" width="600" height="371" seamless frameborder="0" scrolling="no" src="https://todo"></iframe>
```

Comment:
These `todo`s need to be resolved before merging. They're causing the iframes to fail.

![image](https://user-images.githubusercontent.com/1120896/68703167-ea958480-053e-11ea-8a49-488b374927ea.png)